xblock/runtime.py

```diff
@@ -1211,10 +1212,22 @@ class Mixologist(object):
     """
     def __init__(self, mixins):
```

Comment:
Maybe we should make a new param for the paths instead of overloading `mixins` to be a list of either kind?

Reply:
I considered that but went with this since the order of the mixins matters.

Reply:
Ah, I see. Works for me then.